source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -489,12 +489,35 @@ export interface IVersionedChoice {
     value: string;
 }
 
+const NotSetValue = "@@not_set@@";
```

Comment:
Is this because Adaptive.Choice doesn't accept null/undefined as a value?

Reply:
It's because the HTML <choice> tag doesn't.